src/Framework/TestCase.php

```diff
@@ -597,6 +597,9 @@ public function getExpectedExceptionMessageRegExp(): ?string
         return $this->expectedExceptionMessageRegExp;
     }
 
+    /**
+     * @psalm-param class-string<|Throwable> $exception
```

Comment:
Looks broken. Did you mean `class-string<Throwable>` ?

Reply:
Indeed: this is supposed to be a `\`